src/app/Shared/Context/DataContext.tsx

```diff
@@ -1,14 +1,11 @@
 import React, { useState, createContext } from "react";
-import { DbCollection } from "../Model/Client_Isolate.model";
 import { FilterInterface } from "../Model/Filter.model";
 
 interface DataInterface {
-    ZNData: DbCollection;
```

Comment:
I wonder if ZNData was used at all?

Reply:
Only to get the length of the full dataset. Now it is retrieved over the counted isolates endpoint without filter.